docs/docs/guides/metadata.md

```diff
@@ -0,0 +1,118 @@
+---
+comments: true
+---
+
+# Add metadata to your application
```

Comment:
```suggestion
# Context metadata
```
I'd like a shorter title and I think it should be something other than just "metadata" because this isn't about general metadata.

Whatever title you use needs to match the title in mkdocs.yml so that the title in the left frame matches the title displayed for the page.